qutip/qip.py

```diff
@@ -0,0 +1,13 @@
+"""Module replicating the qutip_qip package from within qutip."""
+import sys
+
+try:
+    import qutip_qip
+
+    sys.modules["qutip.qip"] = sys.modules["qutip_qip"]
+    del qutip_qip
+except ImportError:
+    raise ImportError(
+        "'qutip.qip' imports require the 'qutip_qip' package. Install it with "
+        "`pip install qutip-qip` or with `pip install qutip[qip]`."
```

Comment:
```suggestion
        "Importing 'qutip.qip' requires the 'qutip_qip' package. Install it with "
        "`pip install qutip-qip` or with `pip install qutip[qip]`."
```
The phrasing just felt a bit clearer to me this way.

Reply:
Yes, let's drop that in this PR. Could you also add a reference to https://qutip-qip.readthedocs.io/en/stable/index.html?

Reply:
Sure! It's done